managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedger.java

```diff
@@ -648,6 +648,11 @@ default void skipNonRecoverableLedger(long ledgerId){}
      * */
     CompletableFuture<Position> asyncFindPosition(Predicate<Entry> predicate);
 
+    /**
+     * Reverse find position one-entry by one-entry.
+     */
+    CompletableFuture<Position> asyncReverseFindPositionOneByOne(Predicate<Entry> predicate);
```

Comment:
The ManageLedger has been marked as `@InterfaceStability.Stable`. So we need to have a default implementation for the newly added method.